jekyll/_ccie/preview.md

```diff
@@ -170,17 +167,17 @@ Once you have received a Preview license from your account team, the following i
          ]
      }
      ```
-   * region/subnet/sercurity group will be filled automatically
-* Initial Startup
-  * From the main replicated dashboard, click on start to start the application. It will take a few minutes to download all of the necesary docker containers. If you run into an issue where replicted reports that `Failure reported from operator: no such image` just click on start again and it should continue. 
-  * Once the app has started, you can log into CircleCI and start running 2.0 builds!
+10. From the main replicated dashboard, click Start to start the application. It will take a few minutes to download all of the necesary docker containers. If Replicted reports that `Failure reported from operator: no such image` click Start again and it should continue. 
```

Comment:
Typo in `Replicated` and, while not needing to be changed, we call it the Management Console when talking to customers, so calling it Replicated (which I'm not sure it is shown as such anywhere), might be confusing.